test/widgets/content_test.dart

```diff
@@ -6,7 +6,7 @@ void main() {
   testWidgets('Throws if no `PerAccountStoreWidget` ancestor', (WidgetTester tester) async {
     await tester.pumpWidget(
       const RealmContentNetworkImage('https://zulip.invalid/path/to/image.png', filterQuality: FilterQuality.medium));
-    expect(tester.takeException(), isAssertionError);
+    check(tester.takeException()).isA<AssertionError>();
```

Comment:
Ah, thanks for the catch!

The added line `import 'package:checks/checks.dart';` should go in this commit, instead of a later commit.